### ISSUE TITLE
Target Temperature of MashStep is not send to Kettle

### ISSUE THREAD

**Issue by JamFfm** [Manuel83/craftbeerpi4#57]
The target temp of a mashstep is not send to kettle. Therefore the heater does not turn on. Target temp always stay 0 C° 
Setting target temp by kettlecontroll makes the heater turn on when sensortemp is below tarnet temp. This is the reason why I assume it is only sending target temp of step to kettle.

**Comment by Schullebernd** [comment]
The target temperature of the kettle is currently not set in the _extension/mashstep/__init__.py_ on method _on_start()_.
```
async def on_start(self):
        if self.timer is None:
            self.timer = Timer(int(self.props.Timer) *60 ,on_update=self.on_timer_update, on_done=self.on_timer_done)
        self.summary = "Waiting for Target Temp"
        await self.push_update()
```

I added the two lines of code to set the kettle temp and now it works as expected
```
async def on_start(self):
        if self.timer is None:
            self.timer = Timer(int(self.props.Timer) *60 ,on_update=self.on_timer_update, on_done=self.on_timer_done)        
        if self.cbpi.kettle is not None:
            await self.cbpi.kettle.set_target_temp(self.props.Kettle, int(self.props.Temp))
        self.summary = "Waiting for Target Temp"
        await self.push_update()
```

@Manuel83 Should I create a pull request for this?

**Comment by JamFfm** [comment]
is fixed with version 4.0.0.33
